science/imager/Portfile

```
@@ -0,0 +1,98 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           compilers 1.0
+PortGroup           active_variants 1.1
+
+name                imager
+version             202211
+set my_version      [string tolower [clock format [clock scan 2000-[string range ${version} 4 5]-10] -format %b]][string range ${version} 2 3][string range ${version} 6 end]
+supported_archs     arm64 x86_64
+set my_machine      [if {[string match *arm* ${os.arch}]} {list arm64} {list x86_64}]
+categories          science
+platforms           darwin
+maintainers         {u-bordeaux.fr:julien.penguen @penguen}
+license             permissive
+
+description         Radio-interferometry data analysis software
+
+long_description    IMAGER (and its associated visualization tool VIEWER) is a state-of-the-art \
+                    software oriented toward (sub-)millimeter radioastronomical interferometry. \ 
```

CI failed with:
```
Failed to parse file science/imager/Portfile: invalid command name "It"
```
because you had a space after the backslash, which needs to be at the end of the line:
```suggestion
                    software oriented toward (sub-)millimeter radioastronomical interferometry. \
```

---

ok thanks that was the key problem!